modules/EnsEMBL/Web/Document/Element/Acknowledgements.pm

```diff
@@ -70,7 +70,7 @@ sub survey_box {
   my $hub  = $self->hub;
   ## Temporary ad for survey
   my $html = '';
-  my $show_survey = 1;
+  my $show_survey = 0;
   my $is_relevant_page = ($hub->type eq 'Gene' || $hub->type eq 'Transcript' || $hub->type eq 'Variation');
```

Comment:
is that part of your commit? To set the show_survey to 0?

Reply:
Yes - we don't want to show the survey box any more, but we're leaving the code in case we decide to reuse it.

Reply:
I understood what the code is doing but was wondering why it was part of this PR?

Reply:
I don't know - maybe Manoj's sandbox got into a weird state? I think just go ahead and merge it - it's only a one-character change!

Reply:
@ens-ma7 I created a branch from `postreleasefix/97` to make this fix and submitted the PR. I am not sure either why this change is appearing here.

Reply:
Ah ok, that explains all these strange commits, this might cause a git history problem when you merge. And any reason why you create your branch off postreleasefix/97, you should create your branch off master branch

Reply:
@ens-ap5 so if he merges that change to postreleasefix/97 this might go out live anytime someone do a hotfix, is that ok?

Reply:
Yes that's fine - Steve asked me to remove the survey before I went on holiday.

Reply:
> Still this commit shouldn't be there, it might cause a problem when doing mpush.

@ens-ma7 shall I create a new PR with this change?

Reply:
I have now managed to get rid of the other merge commits by rebasing my branch with `postreleasefix/97`. Now the only extra commit is the one that's related to the survey. It is appearing in this branch because it is present in `release/97` but missing in `postreleasefix/97`. I guess it should be fine to merge.